# Service 
geometry_msgs/Pose target_pose

---
# Response of service
int8 num_solutions
ik_service/JointSolutions[8] joint_solutions
bool success
